builtins/amp-ad.js

```diff
@@ -292,7 +292,9 @@ export function installAd(win) {
      */
     getAdCid_() {
       const scope = clientIdScope[this.element.getAttribute('type')];
-      if (!scope) {
+      const consentId = this.element.getAttribute(
+          'data-consent-notification-id');
+      if (!(scope || consentId)) {
         return Promise.resolve();
       }
       return cidForOrNull(this.getWin()).then(cidService => {
```

Comment:
so `userNotificationMananger` is still going to need `cidFor` eventually which is why i thought it was ok to do keep on doing this here.